R/produce_nrs_extract_tests.R

```diff
@@ -0,0 +1,43 @@
+#' Source Extract Tests
+#'
+#' @description Produce a set of tests which can be used by the death (NRS) extract
+#'
+#' This will produce counts of various demographics.
+#' It will also produce various summary statistics for episode date variables.
+#'
+#' @param data new or old data for testing summary flags
+#' (data is from [get_source_extract_path()])
+#' @param max_min_vars variables used when selecting 'min-max' from [calculate_measures()]
+#'
+#' @return a dataframe with a count of each flag
+#' from [calculate_measures()]
+#' @export
+#'
+#' @family extract test functions
+#' @seealso [calculate_measures()]
+produce_nrs_extract_tests <- function(data,
+                                      max_min_vars = c(
+                                        "record_keydate1", "record_keydate2"
+                                      )) {
+  test_flags <- data %>%
+    # use functions to create HB and partnership flags
+    dplyr::arrange(.data$chi) %>%
```

Comment:
```suggestion
```
I don't think we need this arrange here as this is already done in `create_demog_test_flags`

Reply:
Removed this line